src/Integration.Vsix/SLCore/SLCoreLocator.cs

```diff
@@ -31,28 +33,68 @@ internal class SLCoreLocator : ISLCoreLocator
     {
         private const string DefaultPathInsideVsix = "Sloop\\";
         private readonly string basePathInsideVsix;
-        private const string JreSubPath = "jre\\bin\\java.exe";
+        private readonly ISonarLintSettings slSettings;
+        private readonly ILogger logger;
+        private readonly IFileSystem fileSystem;
+        private const string SlCoreJreSubPath = "jre\\bin\\java.exe";
+        private const string WindowsJreSubPath = "bin\\java.exe";
```

Comment:
I think we should require the full path to java.exe and not rely on the bin folder to be present